graphql_compiler/tests/test_data_tools/data/commands.sql

```diff
@@ -0,0 +1,244 @@
+create vertex Species set name = 'Nazgul', uuid = '87425473-4595-2762-9989-378641469948'
```

Comment:
Is this the output from the sql generator script? Add a comment header line or two, something like:
```
# Auto-generated output from <script>, do not edit directly!
# Generated on <datetime> from git revision <git sha>
```